koku/masu/database/report_db_accessor_base.py

```diff
@@ -128,27 +123,22 @@ def _execute_trino_raw_sql_query_with_description(
             context = {}
         if conn_params is None:
             conn_params = {}
-
         if sql_params:
             ctx = self.extract_context_from_sql_params(sql_params)
         elif context:
             ctx = self.extract_context_from_sql_params(context)
         else:
             ctx = {}
-
         sql, bind_params = self.trino_prepare_query(sql, sql_params)
         t1 = time.time()
         trino_conn = trino_db.connect(schema=self.schema, **conn_params)
         LOG.info(log_json(msg="executing trino sql", log_ref=log_ref, context=ctx))
-        try:
-            trino_cur = trino_conn.cursor()
-            trino_cur.execute(sql, bind_params)
-            results = trino_cur.fetchall()
-            description = trino_cur.description
-        except Exception as ex:
-            if attempts_left == 0:
-                LOG.error(log_json(msg="failed trino sql execution", log_ref=log_ref, context=ctx), exc_info=ex)
```

Comment:
Looking at all the calling code, omitting this log message loses the value of `log_ref` that we rely on, so I do think we need to catch the exception here and can't rely fully on the decorator for logging.

From looking at the last 90 days of logs, catching `TrinoQueryError` should be sufficient.
```python
except TrinoQueryError as ex:
    LOG.error(log_json(msg="failed trino sql execution", log_ref=log_ref, context=ctx), exc_info=ex)
    raise
```